internal/addon/manifests/charts/mcoa/charts/metrics/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.enabled }}
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: open-cluster-management:metrics-addon:agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: open-cluster-management:metrics-addon:agent
+subjects:
+  - kind: ServiceAccount
+    name: metrics-addon-sa
+    namespace: {{ .Values.addonInstallNamespace }}
```

Comment:
I've never seen that namespace changing, personally. But I'm not sure if it's hardcoded. I wanted to leave this here because soon we won't need to be in the same namespace. Particularly after we merge the secret manager we should be able to decouple from it.

We still have to comply to the addon configuration and be able to deploy our manifests in different namespaces, no?